gsoc/gsoc2016/ideas_yt.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: default
+title:  "Ideas for yt"
+show_main: false
+ideas_team: yt
+---
+
+All projects on this page are for [yt](https://yt-project.org), an analysis and
```

Comment:
This file doesn't become it's own page, they are all combined into one ideas page: http://openastronomy.org/gsoc/gsoc2016/ideas.html